erigon-lib/kv/tables.go

```diff
@@ -60,6 +60,8 @@ const (
 	ContractCode = "HashedCodeHash"
 )
 
+const Witnesses = "witnesses" // block_num_u64 + "_chunk_" + chunk_num_u64 -> witness
```

Comment:
..witness encoded as - pls attach link you had before to keep reference

Reply:
Not used for now, but I will add the reference.